modules/EmailAddresses/EmailAddress.php

```diff
@@ -121,7 +121,7 @@ public function confirmOptIn()
     {
         global $timedate;
         $date = new DateTime();
-        $this->confirmed_opt_in_date = $date->format($timedate::DB_DATETIME_FORMAT);
```

Comment:
`confirmed_opt_in_date` renamed but still used in SugarEmailAddressTest.php:846, probably this test will failing.